pipelines/concourse.yml

```diff
@@ -289,25 +284,31 @@ jobs:
       passed: [dev-image]
       trigger: true
       params: {format: oci}
-    - get: postgres-image
+    - get: postgres-image-latest
+      resource: postgres-image
       params: {format: oci}
     - get: postgres-image-legacy
       params: {format: oci}
     - get: ci
   - across:
-    - var: postgres-image
-      values: [postgres-image, postgres-image-legacy]
-      max_in_flight: 2
-    - var: enable-containerd
-      values: ["true", "false"]
-      max_in_flight: 2
+    - var: runtime
+      values:
+      - guardian
+      - containerd
+      max_in_flight: all
+    - var: postgres
+      values:
+      - latest
+      - legacy
+      max_in_flight: all
     task: testflight
     image: unit-image
     privileged: true
     timeout: 1h
-    params:
-      CONTAINERD: ((.:enable-containerd))
-      POSTGRES_IMAGE: ((.:postgres-image))
+    params: 
+      RUNTIME: ((.:runtime))
+    input_mapping: 
+      postgres-image: postgres-image-((.:postgres))
```

Comment:
mentioned in standup, but just as a reminder, `input_mapping` doesn't support var interpolation, so you'll have to pass it in as a param and mount both inputs